elevate-cpanel

```diff
@@ -9067,17 +9068,17 @@ sub start ($self) {
 
     $self->_capture_env_variables();    # capture at startup
 
-    if ( $self->getopt('no-leapp') ) {
+    if ( $self->getopt('upgrade-distro-manually') || $self->getopt('no-leapp') ) {
         my $touchfile = PAUSE_ELEVATE_TOUCHFILE;
         WARN( <<~"EOS");
-        'leapp' checks and run are disabled.
+        Automated upgrade of the distro has been disabled.
         Please wait for the file '$touchfile' to be created.
-        At that stage the elevation process is paused then you can upgrade from 7 to 8 on your own.
+        At that stage, the ELevate process is paused, and you can upgrade from 7 to 8 on your own.
 
-        Once the system has been successfully upgraded from 7 to 8 you can then remove that file
-        to let the elevation process continue and update cPanel for the updated distribution.
+        Once the system has been successfully upgraded from 7 to 8, you can then remove that file
```

Comment:
Same as above:

Ubuntu versions need represented here. Maybe we need to check Elevate::OS::instance() or something along those lines and show the appropriate text here. That or we can simply omit the versions here:

At that stage, the ELevate process is paused, and you can upgrade to the next distro version on your own.